docs/imports-exports.md

```diff
@@ -0,0 +1,233 @@
+# Importing and Exporting Data
+
+## Importing Data
+
+### Overview
+
+If you have large amounts of data, it is more efficient to import it into Pilosa instead of using multiple SetBit queries. This library supports importing anything which implements the `Record` interface, such as `Bit`s or `FieldValue`s into an existing frame.
+
+Once you create the frame to import into, you need to create an instance of a struct which implements the `RecordIterator` interace. This library ships with `CSVIterator` which is described further below.
```

Comment:
Once you create the frame in which to import data, you need to create an instance of a struct which implements the `RecordIterator` interface. This library ships with `CSVIterator` which is described further below.

(couple changes here)

also, can we make `CSVIterator` a link to that section and remove "which is described further below"?